src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -421,7 +421,9 @@ export const FIELDS: Readonly<Record<FieldKey, ColumnType>> = {
   // issue.id and project.id are omitted on purpose.
   // Customers should use `issue` and `project` instead.
   [FieldKey.TIMESTAMP]: 'date',
-  [FieldKey.TIME]: 'date',
+  // time is omitted on purpose. time is timestamp rounded down
+  // to 3600 seconds (usually) and presented as seconds since epoch
+  // Customers should almost always use `timestamp`.
```

Comment:
@alxu-sentry you may need to update the `FieldKey` type in this same file.

Reply:
I think I was worried that it'd break something, but apparently it actually breaks it if I *don't* delete it.

Reply:
It's just typescript on frontend. Hard to break if everything is well typed :D